src/regtech_data_validator/data_formatters.py

```diff
@@ -3,32 +3,32 @@
 
 from tabulate import tabulate
 
+more_than_2_fields = ["E2014", "E2015", "W2035", "W2036", "W2037", "W2038", "W2039"]
+
 
 def df_to_download(df: pd.DataFrame) -> str:
```

Comment:
might be ok for MVP; or it might've already been tested to work with all the scenarios; but I think it's worthwhile to see if there is any way we can do df to csv through the library; escaping characters in code doesn't give me the warm and fuzzies; i.e. what if the `validation_desc` had odd number of `"`, what if the field_value had `,`, and / or `"`. etc.